content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -32,6 +32,8 @@ The .NET Tracer supports the following logging libraries:
 
 ## Getting started
 
+Before you begin, 
```

Comment:
Can this be removed? If possible I would like to trim text to make the docs as concise as possible (while still being informative of course)

Reply:
Done